components/count-badge/README.md

```diff
@@ -24,6 +24,21 @@ The `d2l-count-badge` element is a web component to display a number count, eith
 | `type`, default: `count` | String | The type of the badge. Valid options are `"notification"` and `"count"`. Notification badges are orange and count badges are grey. |
 | `max-digits`, default: `2` when `type="notification"` | Number | Optionally specify a digit limit, after which numbers are truncated. Defaults to two for `"notification"` type and no limit for `"count"` type.
 | `hide-zero`, default: `false` | Boolean | Optionally choose not to show the count badge when the number is zero. |
+| `text`, required | String | Descriptive text for the badge which will act as an accessible label and tooltip text when tooltips are enabled. |
+| `tab-stop`, default: `false` | Boolean | Optionally choose to make the badge a tab stop. |

```

Comment:
Just wrapping my head around the need for both `tab-stop` and `has-tooltip`... sometimes we want to be able to tab to a count-badge so the accessibility label can be read, but we _don't_ want a tooltip?

I'm wondering if these two should be combined for a keyboard-only user - basically, if you want to add a tab-stop for screenreader users, then keyboard-only users also benefit from the tooltip.  And the reverse seems to already be true - if you add a tooltip, then you're already adding a tab-stop.

Reply:
The spec mentions we can do a tab stop without a tooltip. I think one case with a tab stop and no tooltip could be portfolio where the text is already showing something similar to what the tooltip would show (ie. overdue evidence). A screenreader may want to still read out "X pieces of evidence" but not show a tooltip there.